tests/test_hooks/conan-center/test_conan-center.py

```diff
@@ -1165,3 +1165,71 @@ def source(self):
                                                              "tools.cross_building(self.settings)"))
         output = self.conan(['export', 'conanfile.py', 'name/version@user/test'])
         self.assertIn("WARN: [TOOLS CROSS BUILDING (KB-H062)] The 'tools.cross_building(self.settings)' syntax in conanfile.py",output)
+
+    def test_strip_root_required_conan_version(self):
+        # no required_conan_version
+        conanfile = textwrap.dedent("""\
+        from conans import ConanFile, tools
+
+        class TestConan(ConanFile):
+            def source(self):
+                tools.get({}, strip_root=True)
+        """)
+        tools.save('conanfile.py', content=conanfile)
+        output = self.conan(['export', '.', 'name/all@user/test'])
+        self.assertIn("WARN: [NO REQUIRED_CONAN_VERSION (KB-H065)] tools.get", output)
+
+        # handle multiline call (for now only two lines)
+        conanfile = textwrap.dedent("""\
+               from conans import ConanFile, tools
+
+               class TestConan(ConanFile):
+                   def source(self):
+                       tools.get({}, 
+                                 strip_root=True)
+               """)
+        tools.save('conanfile.py', content=conanfile)
+        output = self.conan(['export', '.', 'name/version@user/test'])
+        self.assertIn("WARN: [NO REQUIRED_CONAN_VERSION (KB-H065)] tools.get", output)
+
+        # wrong required_conan_version
+        conanfile = textwrap.dedent("""\
+               from conans import ConanFile, tools
+
+               required_conan_version = ">=1.28.0"
+
+               class TestConan(ConanFile):
+                   def source(self):
+                       tools.get({}, strip_root=True)
```

Comment:
```suggestion
                       tools.get({}, strip_root=True)
```

Break the line just to check that for loop.